src/cljdoc/util/datetime.clj

```diff
@@ -9,7 +9,13 @@
 
   Eg: 1st, 2nd, 3rd & 20th."
   [day]
-  (get {1 "st" 2 "nd" 3 "rd"} day "th"))
+  (if (<= 11 day 13)
+    "th"
+    (case (mod day 10)
+      1 "st"
+      2 "nd"
+      3 "rd"
+      "th")))
```

Comment:
haha who wrote this code 😛

Reply:
Hehe! I only noticed because I was looking at builds on the 23th of November!